py/selenium/webdriver/support/expected_conditions.py

```diff
@@ -364,6 +364,21 @@ def _predicate(driver):
     return _predicate
 
 
+def attribute_to_be_include_in_element(locator, attribute_):
```

Comment:
Can we change this to 

```suggestion
def element_attribute_to_include(locator, attribute_):
```

Reply:
@AutomatedTester thanks for the suggestion, have updated the method name